sagemaker-pipelines/tabular/local-mode/framework-processor/sagemaker-pipelines-local-mode-fp.ipynb

```diff
@@ -0,0 +1,945 @@
+{
```

Comment:
<p>Need to make sure this dataset is approved and get it uploaded to the sagemaker-sample-files bucket.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3614/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>